test/functional/p2p_segwit.py

```diff
@@ -923,7 +923,7 @@ def test_witness_block_size(self):
         assert_equal(block.get_weight(), MAX_BLOCK_WEIGHT + 1)
         # Make sure that our test case would exceed the old max-network-message
         # limit
-        assert len(block.serialize()) > 2 * 1024 * 1024
+        assert len(block.serialize()) > 2 * 1024 * 1024 / 10
```

Comment:
please correct MAX_BLOCK_WEIGHT in messages.py to make this test passed

Reply:
Sounds reasonable, instead of dividing I will correct to the same value as defined in consensus https://github.com/BitgesellOfficial/bitgesell/blob/master/src/consensus/consensus.h#L15

Reply:
I mean here, as it used in tests: https://github.com/BitgesellOfficial/bitgesell/blob/c4690c40ab8116b81a27331b31beddf18bc31419/test/functional/test_framework/messages.py#L38

Reply:
yes exactly, correct the value used by the test framework to match BGL consensus settings